SpecFlowTask/SpecFlowTask/StepDefinitions/LangSkillStepDefinitions.cs

```diff
@@ -3,13 +3,15 @@
 using SpecFlowTask.Pages;
 using SpecFlowTask.Utilities;
 using System;
+using System.Reflection.Emit;
 using TechTalk.SpecFlow;
 
 namespace SpecFlowTask.StepDefinitions
 {
     [Binding]
     public class LangSkillStepDefinitions : CommonDriver
     {
+        //Login Functionality
         [Given(@"I logged into the website successfully and navigate to profile page")]
         public void GivenILoggedIntoTheWebsiteSuccessfullyAndNavigateToProfilePage()
```

Comment:
please move the initialise of the chrome driver into the common driver

Reply:
also can you please make use of hooks 
https://docs.specflow.org/projects/specflow/en/latest/Bindings/Hooks.html


https://codebun.com/how-to-implement-hooks-in-specflow-c/

Reply:
Can you also run the add language test 5 times, you will see that you can not add the same language again, you will need to reset the state so that you can add the same language again! this also goes for all the other tests that need the state reset!